app/src/main/java/com/github/libretube/enums/FileType.kt

```diff
@@ -0,0 +1,7 @@
+package com.github.libretube.enums
+
+enum class FileType {
```

Comment:
Maybe that could somewhat be combined with the enum `DownloadType` we already have?

Reply:
`DownloadType` is not used anywhere so we can remove it or rename `FileType` to `DownloadType`.

Reply:
I'd say we rename `FileType` to `DownloadType` because that's what it's currently being used for.